packages/core/src/app.ts

```diff
@@ -427,8 +429,6 @@ export class AppBuilder<TreeNames> {
       }
     }
 
-    html.insertStyleLink(html.styles, `assets/${this.app.name}.css`);
```

Comment:
We need to confirm that if this is what we want to do from the parent engine perspective